setup.cfg

```diff
@@ -51,35 +51,40 @@ keywords =
 
 [options]
 zip_safe = True
-install_requires = 
+install_requires =
+    appdirs ~= 1.4
```

Comment:
Some of the deps do not support Python 3.5 so we will need to drop support for Python <3.6.

Reply:
We already have since we introduced f-strings.